.github/workflows/build-and-test.yml

```diff
@@ -63,11 +63,20 @@ jobs:
       destination: ${{ matrix.platform.destination }}
       resultBundle: ${{ format('TestApp-{0}-{1}.xcresult', matrix.platform.name, matrix.config) }}
       artifactname: ${{ format('TestApp-{0}-{1}.xcresult', matrix.platform.name, matrix.config) }}
+  buildandtest_linux:
```

Comment:
nit but can we call this `package_tests_linux` to have it in line with the other job?